src/main/java/no/unit/nva/cristin/projects/ProjectsWrapper.java

```diff
@@ -0,0 +1,17 @@
+package no.unit.nva.cristin.projects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public class ProjectsWrapper {
+
+    @JsonProperty("@context")
+    public String context;
```

Comment:
For later: annotate the fields with `@JsonProperty` (name is not necesary) to be evident that we are using them for serialization and make them private with getters and setters